src/ccalls.jl

```diff
@@ -7,7 +7,7 @@ function GET_FROM_GAP(ptr::Ptr{Cvoid})::Any
 end
 
 function EvalStringEx( cmd :: String )
-    res = ccall( :GAP_EvalString, MPtr, 
+    res = ccall( :GAP_EvalString, Any,
```

Comment:
Could you explain why `Any` is used here instead of `GapObj`? Perhaps to deal with cases where an FFE is returned? In that case, perhaps `GAPInputType_internal` would be more appropriate?

Reply:
Using `GapObj` there would actually do the same as `Any`, and so would object. They might do more type checking (I am not sure, I will have to check), but they all expect the C function to return a valid `jl_value_t*`, so I decided to not do it.